core/base/document.php

```diff
@@ -1151,14 +1151,16 @@ public function delete() {
 	}
 
 	/**
-	 *
 	 * @since 3.6.0
 	 *
 	 * @param array $config
-	 *
 	 * @param array $map_old_new_post_ids
+	 *
+	 * @deprecated 3.8.0 Use `::updated_on_import_replace_dynamic_content()` instead.
 	 */
 	public static function on_import_replace_dynamic_content( $config, $map_old_new_post_ids ) {
+		Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_function( __METHOD__, '3.8.0', __CLASS__ . '::updated_on_import_replace_dynamic_content()' );
+
 		foreach ( $config as &$element_config ) {
```

Comment:
Refer this suggestion to all the lines from 1164-1174
Maybe use the newly created function instead of duplicating the code?
```suggestion
		$data = [ 'post_ids' => $map_old_new_post_ids, ];
		return static::updated_on_import_replace_dynamic_content( $config, $data ); 
```